stable/prometheus-operator/templates/prometheus/prometheus.yaml

```diff
@@ -64,9 +64,7 @@ spec:
   configMaps:
 {{ toYaml .Values.prometheus.prometheusSpec.configMaps | indent 4 }}
 {{- end }}
-{{- if .Values.global.rbac.create }}
```

Comment:
why did you remove this condition?  It could break the chart when rbac is false and serviceAccount does not exist (it'll not be created by the chart, see here https://github.com/helm/charts/blob/d2e406c9958c6f2275a0243848c0ed0bf2e6bb92/stable/prometheus-operator/templates/prometheus/serviceaccount.yaml

Reply:
When rbac is `false` then `default` account, which should afaik exist at every circumstances (even when RBAC is turned off via k8s flag), is used. https://github.com/helm/charts/blob/d2e406c9958c6f2275a0243848c0ed0bf2e6bb92/stable/prometheus-operator/templates/_helpers.tpl#L62